tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -47,7 +49,12 @@ preferably, RStudio loaded on your computer.
 
 ### Data to Download
 
-{% include/dataSubsets/_data_Imaging-Spec-Data-H5.html %}
+{% include/dataSubsets/_data_Imaging-Spec-Data-H5-2020.html %}
+
+*Remember* that the example dataset included here only has 1 out of every 4 bands
+included in a full NEON hyperspectral dataset (this substantially reduces size!). When 
+we refer to bands in this tutorial, we will note both the band numbers for this 
```

Comment:
Whoops - this used to say that we will refer to both the subset bands and the associated bands in the full dataset, but I removed that information from this tutorial because it became cluttered and confusing.